CHANGELOG.md

```diff
@@ -190,6 +190,14 @@ To fix the error, import `node_modules/govuk-frontend/govuk/base` first. For exa
 
 This change was introduced in [pull request #2455: Remove 'base' import from files in `core` and `overrides` layers](https://github.com/alphagov/govuk-frontend/pull/2455).
 
+#### Import "template" Sass file from the `objects` layer instead of the `core` layer
+
+If you import the "template" Sass file from the `core` layer, you might see the error `File to import not found or unreadable: template` when compiling your Sass.
```

Comment:
Style guide says to only use double quotes for direct quotation (like from what someone said or wrote), and to use single quotes otherwise.

```suggestion
If you import the 'template' Sass file from the `core` layer, you might see the error `File to import not found or unreadable: template` when compiling your Sass.
```